andela/AndelaEatsUI
# #162409411 Reset orders

#### What does this PR do?

- Resets orders when the `Reset Order` button is clicked
- Resets meal items selected when a different day is selected

#### Description of Task to be completed?

- Implement resetting orders when the reset button is clicked
- Implement resetting meal items when a different day is selected

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `bg-fix-reset-orders-162409411`
- run these commands `npm run build` and `npm run dev` 
- Log in as a `User`, select different meals for a given day, then click on the `RESET ORDER` button
- Select meals for a day, then click on a different day
- In both cases, the pre-selected choices should be cleared.

#### What are the relevant pivotal tracker stories?
- [#162409411](https://www.pivotaltracker.com/story/show/162409411)


#### Background Context

N/A

#### Screenshots (If Applicable)

N/A
